### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by nexryai**
Added Japanese translation to this great extension :)

### CODE REVIEW

File: src/_locales/ja/messages.json [hunk 1]
```diff
@@ -0,0 +1,723 @@
...
+    "options_disable_hyperlink_auditing": {
+        "message": "クリックしたリンクを追跡されないようにする  (\"hyperlink auditing\")",
+        "description": "Checkbox label found on the general settings page"
+    },
+    "options_disable_network_prediction": {
+        "message": "プリフェッチを無効にする",
+        "description": "Checkbox label found on the general settings page"
+    },
+    "options_disable_floc": {
+        "message": "Federated Learning of Cohorts (FLoC)を無効化する",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:417]: Missing "Google's".

File: src/_locales/ja/messages.json [hunk 2]
```diff
@@ -0,0 +1,723 @@
...
+    "options_domain_filter_user": {
+        "message": "ユーザー定義",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "popup_info_firstparty_protections": {
+        "message": "リンクトラッキングプロテクションを有効にする",
+        "description": "Clickable status header text in the popup. Shown when Privacy Badger's link tracking protection is active (for example, on facebook.com)."
+    },
+    "popup_info_firstparty_description": {
+        "message": "一部のウェブサイトでは、$LINK_START$あなたがリンクをクリックしてウェブサイトを離れるたびに、リンクトラッキングを使用してあなたを追跡しています。$LINK_END$ Privacy Badgerは、このウェブサイト上のそのようなリンクからトラッキングを削除します。",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:429]: Just checking, is `$LINK_END$` in the appropriate place?

File: src/_locales/ja/messages.json [hunk 3]
```diff
@@ -0,0 +1,723 @@
...
+                "example": "15"
+            },
+            "domain": {
+                "content": "$2",
+                "example": "example.com"
+            }
+        }
+    },
+    "share_base_message": {
+        "message": "Privacy Badger (privacybadger.org) は、目に見えないトラッカーを自動的に学習してブロックするブラウザ拡張機能です。Privacy Badgerは、オンラインであなたの権利を守るために戦う非営利団体、Electronic Frontier Foundationによって作られています。",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:658]: "Electronic Frontier Foundation" is translated in two other messages, but not in this one.

File: src/_locales/ja/messages.json [hunk 4]
```diff
@@ -0,0 +1,723 @@
...
+                "content": "$2",
+                "example": "<a href='https://privacybadger.org/#What-is-a-third-party-tracker'>"
+            },
+            "link_end": {
+                "content": "</a>"
+            }
+        }
+    },
+    "popup_instructions_no_trackers": {
+        "message": "$LINK_START$トラッカー$LINK_END$は検出されませんでした。",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:458]: Remove the trailing period?

File: src/_locales/ja/messages.json [hunk 5]
```diff
@@ -0,0 +1,723 @@
...
+    "tooltip_allow": {
+        "message": "スライダーを右に動かすと、ドメインが許可されます",
+        "description": ""
+    },
+    "intro_privacy_note": {
+        "message": "Privacy Badgerは、ユーザーが共有することを選択しない限り、ユーザーのブラウジングに関するデータを共有することは絶対にありません。",
+        "description": "Intro page paragraph."
+    },
+    "options_domain_list_trackers": {
+        "message": "Privacy Badgerはこれまでに、 $COUNT$個の潜在的な $TRACKER_LINK_START$トラッキングドメイン$TRACKER_LINK_END$ をブロックしました。",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:487]: Remove trailing period?

File: src/_locales/ja/messages.json [hunk 6]
```diff
@@ -0,0 +1,723 @@
...
+    "intro_not_an_adblocker_paragraph": {
+        "message": "Privacy Badgerは、3つの異なるウェブサイトで同じトラッカーが見つかるとブロックを開始します。スリーストライクでアウト!",
+        "description": "Intro page paragraph."
+    },
+    "options_domain_status_filter": {
+        "message": "ステータスで絞り込み:",
+        "description": "Label for a dropdown control on the Tracking Domains options page tab."
+    },
+    "report_text": {
+        "message": "そのエラーについて簡単に説明します。",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:516]: Does this mean something like, "please describe the problem [you experienced on a website] in the text form located below this message"?

File: src/_locales/ja/messages.json [hunk 7]
```diff
@@ -0,0 +1,723 @@
...
+    "share_button_title_facebook": {
+        "message": "Facebookで共有",
+        "description": "Text that comes up when you hover over the social sharing buttons on the intro page."
+    },
+    "report_fail": {
+        "message": "すみません。何らかの問題が発生しました。",
+        "description": ""
+    },
+    "social_tooltip_pb_has_replaced": {
+        "message": "Privacy Badger has replaced this $BUTTON$ button",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:564]: Untranslated.

File: src/_locales/ja/messages.json [hunk 8]
```diff
@@ -0,0 +1,723 @@
...
+        "description": "Tooltip shown over a replaced social button. For example, \"Privacy Badger has replaced this Facebook Like button\". See also the widget_placeholder_pb_has_replaced message.",
+        "placeholders": {
+            "button": {
+                "content": "$1",
+                "example": "Facebook Like"
+            }
+        }
+    },
+    "widget_placeholder_pb_has_replaced": {
+        "message": "Privacy Badgerは、 $START_ANCHOR_TAG$この $WIDGET$ ウィジェット$END_ANCHOR_TAG$ を置き換えました。",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:574]: Remove trailing period?

File: src/_locales/ja/messages.json [hunk 9]
```diff
@@ -0,0 +1,723 @@
...
+                "example": "</a>"
+            }
+        }
+    },
+    "allow_once": {
+        "message": "今回だけ許可",
+        "description": "Button in the placeholder shown in place of certain third-party (video, audio, commenting) widgets. Clicking it will allow the third-party widget to load this one time only."
+    },
+    "allow_on_site": {
+        "message": "このサイトでは常に許可",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:596]: This and the previous message (`allow_once`) are buttons the user can click to activate a replaced widget. Just checking, is the wording appropriate for buttons?

File: src/_locales/ja/messages.json [hunk 10]
```diff
@@ -0,0 +1,723 @@
...
+    "report_text": {
+        "message": "そのエラーについて簡単に説明します。",
+        "description": ""
+    },
+    "intro_donate1": {
+        "message": "Privacy Badger を使用すると、 <a href='https://www.eff.org/' target='_blank'>EFF（電子フロンティア財団）</a> 他の何百万人ものユーザーと一緒になって、プライバシーのために闘うことになります。私たちはオンラインであなたの権利を守るために戦う非営利団体です。ご参加ありがとうございます!",
+        "description": "Part of the 'donate' section on the intro page."
+    },
+    "options_domain_search_tooltip": {
+        "message": "ドメインを除外するには、検索語の前に \"-\"を付けます。例えば、 \".co -.com\" と入力すると、.co と .co.uk は表示されますが、.com は除外されます。",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:524]: Should this use 「」 quotes?

**Review by ghostwords — COMMENTED** [review]
Thank you for your contribution! Please see my feedback inline below.

File: src/_locales/ja/messages.json [hunk 11]
```diff
@@ -0,0 +1,723 @@
+{
+    "name": {
+        "message": "Privacy Badger",
+        "description": "Privacy Badger"
+    },
+    "description": {
+        "message": "Privacy Badgerは、目に見えないトラッカーを自動的に学習してブロックします。",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json:7]: ok I'll fix it!

File: src/_locales/ja/messages.json [hunk 12]
```diff
@@ -0,0 +1,723 @@
...
+    "popup_special_page_paragraph": {
+        "message": "Privacy Badgerは、このような特殊なページでは機能しません。他の場所でブラウズしてみてください。",
+        "description": "Shown in the popup for special browser pages such as the New Tab page and 'about:' pages."
+    },
+    "popup_disabled_site_header": {
+        "message": "Privacy Badgerはこのサイトでは無効になっています",
+        "description": "Shown in the popup on disabled sites."
+    },
+    "disabled_sites": {
+        "message": "保護を無効化するサイト",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json:682]: It is conveyed that way. There should be no problem with this expression!

File: src/_locales/ja/messages.json [hunk 13]
```diff
@@ -0,0 +1,723 @@
...
+    "invalid_json": {
+        "message": "無効なJSONファイルです。",
+        "description": ""
+    },
+    "options_remove_origin_confirm": {
+        "message": "本当にPrivacy Badgerからこのドメインを削除しますか？",
+        "description": "Confirmation shown when you click to remove a domain from Privacy Badger's tracking domains list on the options page."
+    },
+    "badger_status_noaction": {
+        "message": "$DOMAIN$ はトラッキングに使用されていません",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: I will fix it!

File: src/_locales/ja/messages.json [hunk 14]
```diff
@@ -0,0 +1,723 @@
...
+    "intro_donate_subheading": {
+        "message": "私たちのツールに寄付をし、共有することで、私たちをサポートしてください。",
+        "description": "Shown below the Donate button on the intro page."
+    },
+    "intro_not_an_adblocker": {
+        "message": "Privacy Badgerは広告ブロッカーではありません。",
+        "description": "Intro page paragraph heading."
+    },
+    "report_success": {
+        "message": "Thanks! We'll get to the bottom of it.",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: Oops!
I forgot to translate it

File: src/_locales/ja/messages.json [hunk 15]
```diff
@@ -0,0 +1,723 @@
...
+    "domain_slider_cookieblock_tooltip": {
+        "message": "Cookieをブロック",
+        "description": "Tooltip shown when you hover over the center part of a domain slider"
+    },
+    "domain_slider_allow_tooltip": {
+        "message": "許可",
+        "description": "Tooltip shown when you hover over the rightmost part of a domain slider"
+    },
+    "feed_the_badger_title": {
+        "message": "Privacy Badgerの既定の設定を復元します",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: ok I fixed it.

File: src/_locales/ja/messages.json [hunk 16]
```diff
@@ -0,0 +1,723 @@
...
+    "options_domain_filter_block": {
+        "message": "ブロック済み",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "intro_welcome": {
+        "message": "Privacy Badgerは、目に見えないトラッカーをブロックするように自動的に学習します。その方法をご覧ください。",
+        "description": "Intro page welcome paragraph."
+    },
+    "error_input": {
+        "message": "What's Wrong?",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: I forgot to translate here also...

File: src/_locales/ja/messages.json [hunk 17]
```diff
@@ -0,0 +1,723 @@
...
+    "import": {
+        "message": "インポート",
+        "description": ""
+    },
+    "options_widget_replacement_tab": {
+        "message": "ウィジェットの置き換え",
+        "description": "Options page tab heading"
+    },
+    "options_widget_replacement_desc": {
+        "message": "ソーシャルボタンやその他の潜在的に有用な（ビデオ、オーディオ、コメント）ウィジェットをブロックする場合、Privacy Badgerはそれらをクリックで有効になるプレースホルダーに置き換えることができます。",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: This is a bit confusing wording and I would change it to a clearer wording.

File: src/_locales/ja/messages.json [hunk 18]
```diff
@@ -0,0 +1,723 @@
...
+    "options_domain_search": {
+        "message": "ドメインで検索:",
+        "description": "Label for a text input box on the Tracking Domains options page tab."
+    },
+    "options_learn_setting": {
+        "message": "あなたのブラウジングから学習して未知のトラッカーをブロックする",
+        "description": "Checkbox label on the general settings page"
+    },
+    "local_learning_warning": {
+        "message": "学習機能を有効にすると、フィンガープリントのリスクが高まりウェブサイトが個人をより識別しやすくなる場合があります。",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: Yes. The meaning is fine. I removed the last period!

File: src/_locales/ja/messages.json [hunk 19]
```diff
@@ -0,0 +1,723 @@
...
+    "learning_prompt_text1": {
+        "message": "Privacy Badgerは、デフォルトであなたのブラウジングから学習することはありません。",
+        "description": "First part of a prompt to visit the blog."
+    },
+    "learning_prompt_text2": {
+        "message": "詳しくは以下をクリックしてください。",
+        "description": "Second part of a prompt to visit the blog."
+    },
+    "learning_prompt_button": {
+        "message": "Privacy Badgerの変化について",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: It was a misrepresentation and I have corrected it.

File: src/_locales/ja/messages.json [hunk 20]
```diff
@@ -0,0 +1,723 @@
...
+    "options_incognito_setting": {
+        "message": "プライベートウィンドウでも学習機能を有効にする",
+        "description": "Checkbox label on the general settings page"
+    },
+    "options_incognito_warning": {
+        "message": "プライベートウィンドウで学習機能を有効にすると、プライベートウィンドウでの閲覧履歴の痕跡がコンピュータに残ることがあります。デフォルトでは、Privacy Badgerはプライベートウィンドウで既に知っているトラッカーをブロックしますが、新しいトラッカーについては学習しません。プライベートウィンドウで多くのブラウジングを行う場合、このオプションを有効にするとよいでしょう。",
+        "description": "Tooltip on the general settings page"
+    },
+    "learning_prompt_text1": {
+        "message": "Privacy Badgerは、デフォルトであなたのブラウジングから学習することはありません。",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: I would amend to a more correct wording!

File: src/_locales/ja/messages.json [hunk 21]
```diff
@@ -0,0 +1,723 @@
...
+    "report_cancel": {
+        "message": "キャンセル",
+        "description": ""
+    },
+    "tooltip_block": {
+        "message": "スライダーを左に動かすと、ドメインがブロックされます",
+        "description": ""
+    },
+    "options_enable_dnt_checkbox": {
+        "message": "ウェブサイトに \"<a href='https://globalprivacycontrol.org/' target='_blank'>Global Privacy Control</a>\" と \"<a href='https://www.eff.org/issues/do-not-track' target='_blank'>Do Not Track</a>\" 信号を送信する",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: "Global Privacy Control" should not be translated. I will fix the quotation marks!

File: src/_locales/ja/messages.json [hunk 22]
```diff
@@ -0,0 +1,723 @@
...
+    "intro_learns_paragraph": {
+        "message": "Privacy Badgerはリストに基づいてブロックする代わりに、ウェブサイトの挙動を監視し自動的にトラッカーを発見します。",
+        "description": "Intro page paragraph"
+    },
+    "intro_beyond_ads": {
+        "message": "卑劣なトラッカーを検出",
+        "description": "Intro page paragraph heading."
+    },
+    "intro_beyond_ads_paragraph": {
+        "message": "目に見えない追跡は、さまざまな方法で行われます。 広告は氷山の一角にすぎません。 Privacy Badgerは、データの共有と販売を拒否するための<a href='https://globalprivacycontrol.org/' target='_blank'>グローバル プライバシー コントロール信号</a>と、あなたを追跡しないようにウェブサイトに指示するための<a href='https://www.eff.org/issues/do-not-track' target='_blank'>Do Not Track信号</a>を送信します。ウェブサイトがあなたの希望を無視した場合、Privacy Badgerはそれらの卑劣なトラッカーをブロックすることを学習します。",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: It was missing, so I fix it!

File: src/_locales/ja/messages.json (hunk 1 above)
Comment (nexryai) [reply src/_locales/ja/messages.json]: It was missing, so I fix it!

File: src/_locales/ja/messages.json [hunk 23]
```diff
@@ -0,0 +1,723 @@
...
+    "intro_donate1": {
+        "message": "Privacy Badger を使用すると、 <a href='https://www.eff.org/' target='_blank'>EFF（電子フロンティア財団）</a> 他の何百万人ものユーザーと一緒になって、プライバシーのために闘うことになります。私たちはオンラインであなたの権利を守るために戦う非営利団体です。ご参加ありがとうございます!",
+        "description": "Part of the 'donate' section on the intro page."
+    },
+    "options_domain_search_tooltip": {
+        "message": "ドメインを除外するには、検索語の前に \"-\"を付けます。例えば、 \".co -.com\" と入力すると、.co と .co.uk は表示されますが、.com は除外されます。",
+        "description": "Tooltip for an \"information\" icon next to the domain search input on the Tracking Domains options page tab."
+    },
+    "intro_disable_button": {
+        "message": "Privacy Badgerがページを壊していると思われる場合（ビデオが再生されないなど）、[このサイトでは保護を無効にする]をクリックすると、そのサイトのPrivacy Badgerをオフにすることができます。",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: I will fix the quotation marks. Thanks for pointing that out!

File: src/_locales/ja/messages.json (hunk 2 above)
Comment (nexryai) [reply src/_locales/ja/messages.json:429]: OK 
I think there is no problem.

File: src/_locales/ja/messages.json (hunk 4 above)
Comment (nexryai) [reply src/_locales/ja/messages.json]: ok!
I deleted it

File: src/_locales/ja/messages.json (hunk 6 above)
Comment (nexryai) [reply src/_locales/ja/messages.json]: I made a mistake!
I will correct that.

File: src/_locales/ja/messages.json (hunk 7 above)
Comment (nexryai) [reply src/_locales/ja/messages.json]: I forgot to translate it. I will correct it.

File: src/_locales/ja/messages.json (hunk 9 above)
Comment (nexryai) [reply src/_locales/ja/messages.json]: This is a bit confusing wording and I would amend it to the proper wording

File: src/_locales/ja/messages.json (hunk 10 above)
Comment (nexryai) [reply src/_locales/ja/messages.json]: Thank you.
I'll fix it.

File: src/_locales/ja/messages.json (hunk 11 above)
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json:7]: ok 
I'll fix it!

File: src/_locales/ja/messages.json [hunk 24]
```diff
@@ -0,0 +1,723 @@
+{
+    "name": {
+        "message": "Privacy Badger",
+        "description": "Privacy Badger"
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json]: Sorry! 
I'll fix it!

File: src/_locales/ja/messages.json (hunk 2 above)
Comment (ghostwords) [reply src/_locales/ja/messages.json:429]: The period at the end of the sentence is part of the link. Do we want that?

File: src/_locales/ja/messages.json [hunk 25]
```diff
@@ -561,7 +561,7 @@
         "description": ""
     },
     "social_tooltip_pb_has_replaced": {
-        "message": "Privacy Badger has replaced this $BUTTON$ button",
+        "message": "Privacy Badgerは、この$BUTTON$ボタンを置き換えました。",
```
Comment (ghostwords) [inline src/_locales/ja/messages.json:564]: Remove trailing period?

File: src/_locales/ja/messages.json [hunk 26]
```diff
@@ -589,11 +589,11 @@
         }
     },
     "allow_once": {
-        "message": "今回だけ許可",
+        "message": "今回だけ読み込む",
         "description": "Button in the placeholder shown in place of certain third-party (video, audio, commenting) widgets. Clicking it will allow the third-party widget to load this one time only."
     },
     "allow_on_site": {
-        "message": "このサイトでは常に許可",
+        "message": "このサイトでは常に読み込む",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json:596]: ok!
I will update the `options_widget_site_exceptions_label` .

File: src/_locales/ja/messages.json (hunk 25 above)
Comment (nexryai) [reply src/_locales/ja/messages.json:564]: Thank you.
I will fix it

File: src/_locales/ja/messages.json [hunk 27]
```diff
@@ -0,0 +1,723 @@
...
+    "popup_disabled_site_header": {
+        "message": "Privacy Badgerはこのサイトでは無効になっています",
+        "description": "Shown in the popup on disabled sites."
+    },
+    "disabled_sites": {
+        "message": "保護を無効化するサイト",
+        "description": "Options page tab heading"
+    },
+    "disabled_for_these_domains": {
+        "message": "<p>Privacy Badger は次のサイトで無効になっています。つまり、Privacy Badger はここにリストされたサイトにアクセスしても何もブロックせず、Do Not Track や Global Privacy Control のシグナルを送信しません。</p><p>Privacy Badgerがページを壊していると思う場合、または特定のサイトがデータを共有または販売することを許可したい場合は、下のボックスにそのページのドメインを入力して、「ドメインを追加」ボタンをクリックしてください。</p>",
```
[the remark this answers is not in the document]
Comment (nexryai) [reply src/_locales/ja/messages.json:686]: "Global Privacy Control" should not be translated.  I will fix it.

File: src/_locales/ja/messages.json (hunk 2 above)
Comment (nexryai) [reply src/_locales/ja/messages.json:429]: Hmm...
Certainly the label on this link seems too long.
I will improve it!

Comment (ghostwords) [reply src/_locales/ja/messages.json]: Everything between `$LINK_START$` and `$LINK_END$` will be a hyperlink. Does the translation say something like, "use link tracking to follow you" as the link text? That is what the original message says.

Comment (nexryai) [reply src/_locales/ja/messages.json]: OK!
I'll fix it to be that way.

**Comment by ghostwords** [comment]
I just merged your translations and added them to [our project on Transifex](https://www.transifex.com/eff/privacy-badger/) (a translation platform). Thank you again for your contributions!

If you are interested, you could help translate the [Privacy Badger homepage](https://privacybadger.org/). Homepage translations are [hosted on Weblate](https://hosted.weblate.org/projects/privacy-badger/) (another translation platform).

**Comment by ghostwords** [comment]
Hi @nexryai, your translations are now live, part of Privacy Badger version 2022.9.27.

**Pull request merged**